matchzoo/datapack.py

```diff
@@ -48,11 +47,9 @@ def __init__(self,
             pre-processing stage.
         :param columns: List of column names of the :attr:`data`
             variable.
-        :param columns_mapping: List of column names of the
-            :attr:`mapping` variable.
         """
         self._dataframe = pd.DataFrame(data, columns=columns)
-        self._mapping = pd.DataFrame(mapping, columns=columns_mapping)
+        self._mapping = mapping
```

Comment:
Here, I think the `self._dataframe` should store pair information such as '(qid0, did0, label)', and `self._mapping` store the 'id: content'. In this way, we can build instances based on `self._dataframe`, then, for each id in the instance (e.g., (qid0, qid1, 1)), the content can be easily obtained from the `self._mapping` as the `self._mapping` is dict.